rnd/autogpt_builder/README.md

```diff
@@ -18,8 +30,10 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
 
+For subsequent runs, you do not have to `install` again.
```

Comment:
"Unless you update to latest git" or similar. Probably better worded than mine 😅

Reply:
I was considering to place this line at the beginning.
but I think most people who use this setup guide are using it for the first time. so installation would almost always be needed, except for subsequent runs.

Reply:
Yeah but since it’s in major development we regularly are adding packages which won’t work unless they re run install